lib/Controller/Campaign.php

```diff
@@ -282,12 +282,12 @@ public function grid(Request $request, Response $response)
             $campaign->includeProperty('buttons');
             $campaign->buttons = [];
 
-            // Schedule Now
+            // Schedule
             if ($this->getUser()->featureEnabled('schedule.now') && $campaign->type === 'list') {
```

Comment:
Should we remove the `schedule.now` feature and then test `schedule.add` instead?

Reply:
That's done now in the commit below